website/content/partials/faq/client-count/tools.mdx

```diff
@@ -2,7 +2,7 @@
 
 @include 'alerts/auditor-deprecated.mdx'
 
-The [Vault auditor tool](/vault/tutorials/monitoring/usage-metrics#vault-auditor-tool)
+The [Vault auditor tool](/vault/docs/concepts/client-count/client-usage#vault-auditor-tool)
```

Comment:
```suggestion
The Vault auditor tool
```

We should delete the link since it doesn't go anywhere useful and we don't support the tool anymore (and it looks like we haven't in a long time)